api/lib/domain/read-models/participant-results/AssessmentResult.js

```diff
@@ -12,6 +12,7 @@ class AssessmentResult {
     participationResults,
     isCampaignMultipleSendings,
     isOrganizationLearnerActive,
+    isTargetProfileAllowReset,
```

Comment:
Je propose de renommer ce champ pour qu'il soit un peu plus parlant (au moins pour moi)

```suggestion
    isTargetProfileResetAllowed,
```